omod/src/main/webapp/master-single-page-application.jsp

```diff
@@ -7,14 +7,14 @@
     <base href="${requestScope.openmrsBaseUrlContext}${requestScope.spaBaseUrlContext}${'/'}" />
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <meta name="importmap-type" content="systemjs-importmap">
-    <link rel="preload" href="${cookie['import-map-override-url'] == null ? '/import-map.json' : cookie['import-map-override-url'].getValue()}" as="fetch" crossorigin="anonymous" />
-    <script type='systemjs-importmap' src="${cookie['import-map-override-url'] == null ? '/import-map.json' : cookie['import-map-override-url'].getValue()}"></script>
-    <script src="https://cdn.jsdelivr.net/npm/import-map-overrides@1.7.2/dist/import-map-overrides.min.js"></script>
-    <script src="https://cdnjs.cloudflare.com/ajax/libs/systemjs/4.1.0/system.min.js"></script>
-    <script src="https://cdnjs.cloudflare.com/ajax/libs/systemjs/4.1.0/extras/amd.min.js"></script>
-    <script src="https://cdnjs.cloudflare.com/ajax/libs/systemjs/4.1.0/extras/named-exports.js"></script>
-    <script src="https://cdnjs.cloudflare.com/ajax/libs/systemjs/4.1.0/extras/named-register.min.js"></script>
-    <script src="https://cdnjs.cloudflare.com/ajax/libs/systemjs/4.1.0/extras/use-default.min.js"></script>
+    <link rel="preload" href="${cookie['import-map-override-url'] == null ? '/frontend/import-map.json' : cookie['import-map-override-url'].getValue()}" as="fetch" crossorigin="anonymous" />
+    <script type='systemjs-importmap' src="${cookie['import-map-override-url'] == null ? '/frontend/import-map.json' : cookie['import-map-override-url'].getValue()}"></script>
+    <script src="/frontend/import-map-overrides@1.7.2/dist/import-map-overrides.min.js"></script>
+    <script src="/frontend/systemjs@4.1.0/system.min.js"></script>
```

Comment:
The switch from `/` to `@` is a good one I think. https://www.jsdelivr.com/ is a good CDN to use for Ampath and others who want to reverse proxy this internal urls to external urls, and jsdelivr [uses @ in its urls](https://www.jsdelivr.com/package/npm/systemjs)